installation/scripts/installer.sh

```diff
@@ -92,13 +91,13 @@ if [ ${LOCAL} ]; then
     COMBO_YAML=$(sed 's/\.minikubeIP: .*/\.minikubeIP: '"${MINIKUBE_IP}"'/g' <<<"$COMBO_YAML")
 fi
 
-kubectl apply -f - <<<"$COMBO_YAML"
-
 echo -e "\nConfiguring sub-components"
 bash ${CURRENT_DIR}/configure-components.sh
 
-echo -e "\nGetting Helm secrets"
-bash ${CURRENT_DIR}/tiller-tls.sh
+echo -e "\nApplying combo yaml"
+kubectl apply -f - <<< "$COMBO_YAML"
+
+echo -e "\nDownloading Helm certificates"
```

Comment:
```suggestion
echo -e "\Setting up Helm certificates"
```

Reply:
This is wrong, we are in fact downloading the certs from the kubernetes secrets to loca files, we do not set them :)